pkg/storaged/utils.js

```diff
@@ -518,6 +520,7 @@ export function get_direct_parent_blocks(client, path) {
         parent = client.lvols[parent].VolumeGroup;
     if (client.vgroups[parent])
         return client.vgroups_pvols[parent].map(function (pv) { return pv.path });
+    // XXX - stratis blockdevs?
```

Comment:
Yeah, what about them?

Reply:
Yes, they need to be included, and creating a filesystem needs to use this function to decide whether to add "netdev" to fstab.

Reply:
Let's not worry about "netdev"... otherwise done.